tasks/section_10_level1.yml

```diff
@@ -98,22 +98,18 @@
       - section10.4
 
   - name: 10.5.1 Lock Inactive User Accounts (check) (Scored)
-    command: grep INACTIVE /etc/login.defs
+    command: bash -c "useradd -D | grep -oP '(?<=^INACTIVE=)-?[0-9]+'"
```

Comment:
Okay. In that case I think using the shell module would be cleaner, since we end up using a shell anyway. @awailly What do you think?